### DISCUSSION TITLE
Unable to push PR

### DISCUSSION THREAD

**Discussion by RonHachmon** [simple-icons/simple-icons#7607] [Help with SVGs] [answered]
im been working a issue #7601 , but i get  a permision problem when i try to push the PR,
has anyone else had this problem?
![image](https://user-images.githubusercontent.com/88894070/179837351-65368b77-214d-425b-bade-9dde9d7f6c42.png)

**Comment by sachinraja** [comment, marked as answer]
You need to fork the repository and push to a branch there.

**Comment by Anmol-Baranwal** [comment]
Unfortunately, this issue is closed & you need to clone the forked repository in order to work, and later pull request the changes you have done.
